docs/cvl/foundry-integration.md

```diff
@@ -112,4 +112,6 @@ you can use `verifyFoundryFuzzTestsNoRevert`. This ignores all code paths that l
 - In Foundry, the `vm.expectRevert` cheatcode can optionally take a specific revert reason. 
 Foundry verifies both that the test function reverted and that it reverted with the specified reason. 
 In contrast, the Prover does not analyze the revert reason. As a result, if the test function reverts for a reason other than 
-the expected one, the test will still be marked as successful.
\ No newline at end of file
+the expected one, the test will still be marked as successful.
+
+- The call trace for the Foundry integration can be hard to read as it is internally implemented as a parametric CVL rule with case splitting on the method `.selector` of each method. This leads to the call trace containing an `if` statement per fuzz test method. When focusing on a particular test it is helpful to use the `--method <FUZZ_TEST_METHOD_NAME>` [flag](https://docs.certora.com/en/latest/docs/prover/cli/options.html#method-method-signature).
```

Comment:
It would be very helpful if we can show an example of the problem and how to resolve it:
1. An exemplary Foundry test Solidity file with two simple and short tests
2. A screenshot of the call trace of these tests showing the parametric rule
3. A command line example showing how to focus on the results of a single test

As is, it is hard to understand the text, and our users always complain that there are not enough examples.

Reply:
Thanks for the feedback - I added screenshots and links to existing jobs to clarify what we mean here. @urikirsh please re-review.